### PR TITLE
JS: published hacl-wasm 1.2.0

### PR DESCRIPTION
**Pull request by victor-dumitrescu**
Small changes made to publish the [hacl-wasm npm package](https://www.npmjs.com/package/hacl-wasm)

### CODE REVIEW

File: bindings/js/Makefile [hunk 1]
```diff
@@ -14,5 +14,7 @@ endif
 all:
 	cp $(KRML_HOME)/krmllib/js/loader.js .
 	cp ../../dist/wasm/*.{wasm,js,html} .
+	cp ../../dist/wasm/layouts.json .
```
Comment (R1kM) [inline bindings/js/Makefile:17]: Why is this now needed?

**Review by R1kM — APPROVED** [review]
Thanks!

File: bindings/js/Makefile (hunk 1 above)
Comment (victor-dumitrescu) [reply bindings/js/Makefile:17]: It was introduced in `dist` after the previous release

**Review by msprotz — APPROVED** [review]
Looks good to me, thanks Victor for taking care of this!

**Pull request merged**